File: GO-BASIC/01.first-steps-in-coding/07.projects-creation/projects-creation.go
package main

import "fmt"

func main() {
	var nameOfArchitect string
	var projectNumbers int
	
	fmt.Scanln(&nameOfArchitect)
	fmt.Scanln(&projectNumbers)

	var hoursRequired int = projectNumbers * 3

	fmt.Printf("The architect %s will need %d hours to complete %d project/s.", nameOfArchitect, hoursRequired, projectNumbers)
	
}

File: C++/C++Basic/01.First-Steps-In-Coding-Lab/FirstStepsInCoding/projectsCreation/projectsCreation.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	string name;
	cin >> name;

	int numberOfProjects;
	cin >> numberOfProjects;

	int hours = numberOfProjects * 3;

	cout << "The architect " << name << " will need " << hours << " hours to complete " << numberOfProjects << " project/s.";

	return 0;
}